# Remove -B/usr/bin from linker_bin_path_flag, as it causes a possibly
# incompatible linker from the system to be used.
#
# Fokke Dijkstra, 20190228
diff -ru tensorflow-1.12.0.orig/third_party/gpus/cuda_configure.bzl tensorflow-1.12.0/third_party/gpus/cuda_configure.bzl
--- tensorflow-1.12.0.orig/third_party/gpus/cuda_configure.bzl	2018-11-02 02:35:10.000000000 +0100
+++ tensorflow-1.12.0/third_party/gpus/cuda_configure.bzl	2019-02-28 14:25:50.227358135 +0100
@@ -1316,7 +1316,7 @@
     if should_download_clang:
       cuda_defines["%{linker_bin_path_flag}"] = ""
     else:
-      cuda_defines["%{linker_bin_path_flag}"] = 'flag: "-B/usr/bin"'
+      cuda_defines["%{linker_bin_path_flag}"] = ""
 
     if is_cuda_clang:
         cuda_defines["%{host_compiler_path}"] = str(cc)
